src/archetype_test.mojo

```diff
@@ -65,19 +71,19 @@ def test_archetype_reserve():
 
 
 def test_get_entity():
-    var component1 = ComponentInfo[DType.uint8](id=1, size=4)
-    var component2 = ComponentInfo[DType.uint8](id=2, size=8)
-    var archetype = Archetype[DType.uint8](10, component1, component2)
+    var component1 = ComponentInfo(id=1, size=4)
+    var component2 = ComponentInfo(id=2, size=8)
+    var archetype = Archetype(0, CArr(component1, component2))
 
     var entity = Entity(0, 0)
```

Comment:
Yes, this should be the same in larecs. Do you think I should check for zeros in the constructor? Otherwise I think the issue is only with the test, since in reality you never create entities on your own.

Reply:
In Arche, the zero entity is created on world creation, and the entity pool should never return it, as it can't be released/recycled. So if your test gives you the zero entity (or rather, an entity with ID==0), you obviously don't have this yet.

The zero entity is the only one that can be created by the user, via `e = Entity{}`. An uninitialized value of type `Entity` should also equal the zero entity.